packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,255 @@
+
+    const hub = getCurrentHub();
+
+    /**
+     * This function sets Additional Runtime Data which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    const setAdditionalRuntimeData = (scope: Scope): void => {
+      scope.setContext('Runtime', {
+        Name: 'node',
+        // global.process.version return the version of node
+        Version: global.process.version,
+      });
+    };
+
+    /**
+     * This function sets Additional Lambda Parameters which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    const setAdditionalLambdaParameters = (scope: Scope): void => {
+      const remainingTimeInMillisecond: number = this._awsContext.getRemainingTimeInMillis();
+      const executionTime: number = configuredTimeInMilliseconds - remainingTimeInMillisecond;
+
+      scope.setContext('lambda', {
+        aws_request_id: this._awsContext.awsRequestId,
+        function_name: this._awsContext.functionName,
+        function_version: this._awsContext.functionVersion,
+        invoked_function_arn: this._awsContext.invokedFunctionArn,
+        execution_duration_in_millis: executionTime,
+        remaining_time_in_millis: this._awsContext.getRemainingTimeInMillis(),
+      });
+    };
+
+    /**
+     * This function use to generate cloud watch url
+     */
+    const cloudwatchUrl = (): string => {
+      /**
+       * process.env.AWS_REGION -  this parameters given the AWS region
+       * process.env.AWS_LAMBDA_FUNCTION_NAME - this parameter provides the AWS Lambda Function name
+       */
+      return `https://${process.env.AWS_REGION}.console.aws.amazon.com/cloudwatch/home?region=${process.env.AWS_REGION}#logsV2:log-groups/log-group/$252Faws$252Flambda$252F${process.env.AWS_LAMBDA_FUNCTION_NAME}`;
+    };
+
+    /**
+     * This function sets Cloud Watch Logs data which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    const setCloudwatchLogsData = (scope: Scope): void => {
+      scope.setContext('cloudwatch logs', {
+        log_group: this._awsContext.logGroupName,
+        log_stream: this._awsContext.logStreamName,
+        url: cloudwatchUrl(),
+      });
+    };
+
+    /**
+     * This function sets tags which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+
+    const setTags = (scope: Scope): void => {
+      scope.setTag('runtime', `node${global.process.version}`);
+      scope.setTag('transaction', this._awsContext.functionName);
+      scope.setTag('runtime.name', 'node');
+      scope.setTag('server_name', process.env._AWS_XRAY_DAEMON_ADDRESS || '');
+      scope.setTag('url', `awslambda:///${this._awsContext.functionName}`);
+    };
+
+    const flushTime = this._flushTime;
```

Comment:
Same naming issue as in the `constructor`

Reply:
Changed  _flushTime to _flushTimeout.